WordPress/gutenberg
# Menu Items: Lower the specificity of the fallback background color

## What?

This PR is lowering the specificity of the fallback background color for the menu items, so the color selected in the block settings is honored.

## Why?


This PR solves the background color issue in the submenu items in the navigation block. 
The issue exists on the Varia and its child themes.

## How?

When using the Navigation block with the Varia and its child themes, the background color selected from the block settings is not honored. It is because the specificity of the fallback menu background color is higher than the CSS specificity of the selected block color.

This PR lowers the specificity of the fallback background color for the menu items to honor the user-selected settings.

## Testing Instructions

1. Activate Hever or any other Varia child theme on a site.
2. Open a post or page and add the Navigation block.
3. Add menu items and submenu items.
4. Choose a background color(primary, secondary, foreground) for the navigation block.
5. View the page. The background color should be visible on clocking the submenu.  


## Screenshots or screencast 
| Before |

![menu navigation-issue](https://user-images.githubusercontent.com/21078987/177701927-3c5972d8-bdee-46ec-b497-ac24afcd3235.png)

| After |
![Menu navigation - fixed](https://user-images.githubusercontent.com/21078987/177701959-a48ae47b-d7b7-4f94-85fb-6f1fd4f2ae15.png)
